pre/main.py

```diff
@@ -5,30 +5,35 @@
 config = None
 __bitswan_dev = False
 
-notebook_path = os.environ.get("JUPYTER_NOTEBOOK", "pipelines/main.ipynb")
+notebook_path = os.environ.get("JUPYTER_NOTEBOOK", "main.ipynb")
```

Comment:
Hm? I'm confused how this change relates to the rest of the PR.

Reply:
I had problems with deployment with this path...and it also affects deployments of our clients...but yeah it should've been part of different PR